### ISSUE TITLE
HD Support?

### ISSUE THREAD

**Issue by zpoo32** [spacys/mediathek#19]
Will HD support be added? I can help test if needed. Most songs I've come across are HD or Ultra HD on the official apps, but this plugin only supports up to 256kbps. Thank you!

**Comment by spacys** [comment]
I can add these paramaters as soon as I have the required one. Unfortunately to figuring out the right ons a separate account/contract is necessary (which I do not have).
Maybe you can provide more details.

**Comment by zpoo32** [comment]
I have a family account with a three month free trial. How can I send you info privately?

**Comment by spacys** [comment]
There is no need to send the necessary parameter privately.
You can check this via Chrome -> F12, start the playback and have a look at the network traffic. In there you should see something like: `'bitRate' : 'HIGH'` and maybe some more like: 256kbit

**Comment by zpoo32** [comment]
Actually, the web player doesn't actually support HD and Ultra HD, so it's blanked out, and I checked the code (ok, just searched it, don't know how to really do that kind of thing) and the options in the bitrate script thing was missing HD and Ultra HD entirely... does that mean it's not possible? Or maybe it will need fiddler or something and a lot of work? I don't know anything about doing something like this :/

**Comment by zpoo32** [comment]
I could just give you an account to work with. That's what I meant earlier when I wanted to send you info privately.

**Comment by spacys** [comment]
Thanks for the trust, but honestly please **never** share **your** account credentials with anybody. That's a very high risk. Usually you can buy anything and compromise your account and **you** have to pay for it.
So again, **never** share your credetials. Please.

**Comment by zpoo32** [comment]
I actually used completely fake credentials for everything, even a fake credit card (there was a working bin to generate a credit card with at the time) so I have a completely anonymised family account I can add a member to. You could just make an amazon account (for the US with an IP that isn't blacklisted, for example with expressvpn, not nordvpn (blacklisted, also one of the worst vpn companies in general) or my personal favourite is called 'unlocator' because a free trial needs just an email address) and I can add you to the family, trusting you use a good VPN. No real details will be shared, and I will only get an email address and a name, but those can very easily be faked. I actually physically cannot buy Amazon Music Unlimited HD because it's not available where I live and Amazon only allows credit cards from a matching country. I could make the Amazon account for you, but then there would be some trust on me. Any way though, no personal info is shared on either side, otherwise I wouldn't even think about sharing my credentials, even though I do trust you more than the average internet user because of this plugin you made

**Comment by spacys** [comment]
Feel free to provide the desired code. In the meanwhile I will close this issue.

**Comment by zpoo32** [comment]
oh yeah I completely forgot about this. I used fiddler and found out the actions for selecting quality in desktop player and web player are very different, and they have different names, even internally. The action for selecting HD/Ultra HD quality in the desktop player is selectAudioQualityHD so it's probably just HD because I checked changing it to standard and I got selectAudioQualityStandard. I can just make a new amazon music hd account and send it to you on that forum this app is on if you want to test yourself.
